src/vs/workbench/contrib/chat/common/chatModel.ts

```diff
@@ -196,9 +196,13 @@ export interface IDiagnosticVariableEntry extends IBaseChatRequestVariableEntry,
 	readonly kind: 'diagnostic';
 }
 
+export interface IElementVariableEntry extends Omit<IBaseChatRequestVariableEntry, 'kind'>, IDiagnosticVariableEntryFilterData {
```

Comment:
That `extends`-clause is too adventerous. The `Omit` shouldn't be needed anymore (I cleaned that up a while back) and I don't think diagnostics data makes much sense?

Reply:
ah yes, result of a hasty copy and paste 🤦🏻